frontend/reducers/auth_reducer.js

```diff
@@ -0,0 +1,24 @@
+import merge from 'lodash/merge';
+
+import {
+    RECEIVE_AUTH_CODE
+} from '../actions/auth_actions';
+
+const defaultState = Object.freeze({
+    clientId: '1046169933414-6mfd4o46adn4lcdmk79mue74h8ca3ou9.apps.googleusercontent.com',
```

Comment:
what is this default client ID?

Reply:
Yeah, its the public identifier so that the google knows which app is requesting user data